projects/gui/src/mainwindow.cpp

```diff
@@ -762,16 +762,17 @@ void MainWindow::onGameFinished(ChessGame* game)
 	}
 
 	// save game notation of non-tournament games to default PGN file
-	if (!tab.m_tournament
-	&&  !game->pgn()->isNull()
-	&&  	(  !game->pgn()->moves().isEmpty()   // ignore empty games
-		|| !game->pgn()->result().isNone())) // without adjudication
+    auto pgn = game->pgn();
```

Comment:
This probably should be a reference (auto const &pgn, for example), although I wouldn't make this change unless necessary